subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        cont = self.sub_stuff["cont"]
+        name_base = cont.get_unique_name()
+        names = []
+        for number in xrange(self.config['names_count']):
+            name = ('%s_%d' % (name_base, number))
+            names.append(name)
+            self.sub_stuff['containers'].append(name)  # just in case
```

Comment:
this line you probably do need, otherwise test-container won't get cleaned up after test.